client/store/getters.js

```diff
@@ -1,5 +1,9 @@
 // Store all getters used in project
 export default {
+    /**
+     * Get token for authorization process.
```

Comment:
I think this docstring is wrong. IMO this function does not returns token. It returns `true` when token is not null and `false` when it does. Am I correct?